tests/tests_certificate_runafter.yml

```diff
@@ -0,0 +1,113 @@
+---
+# This approach uses a `run_after:` shell script. This works everywhere,
+# including RHEL/CentOS 7, but is hackish and non-obvious in `getcert list`.
+# tests_certificate.yml covers a cleaner approach.
+- name: test - local setup
+  hosts: 127.0.0.1
+  gather_facts: false
+  tasks:
+    - name: Download current linux-system-roles.certificate
+      git:
+        repo: https://github.com/linux-system-roles/certificate/
+        dest: roles/linux-system-roles.certificate
+        version: master
+        depth: 1
+      become: false
+      # skip this for RHEL downstream tests, which define $BEAKERLIB
+      when:
+        - lookup('env', 'BEAKERLIB') | length == 0
+        - not 'roles/linux-system-roles.certificate/.git/HEAD' is exists
```

Comment:
not sure why there is a failure here - https://152.19.134.199/groups/linuxsystemroles/logs/lsr-citool_cockpit-40-559c584_20211026-014422/artifacts/tests_certificate-FAILED.log - if the role already exists, then why doesn't `roles/linux-system-roles.certificate/.git/HEAD` exist?

Reply:
Oh ouch -- but at least that is a more tangible error than the ones which had inventory errors or boot failures, or the [RHEL 7.9 failure](https://152.19.134.199/groups/linuxsystemroles/logs/lsr-citool_cockpit-40-559c584_20211102-154407/artifacts/) about missing repositories. 

E.g. in [this run](https://152.19.134.199/groups/linuxsystemroles/logs/lsr-citool_cockpit-40-559c584_20211102-151631/artifacts/), it is test_certificate that failed in this way, and the _runafter test passed. My theory is that this ran the test in the opposite order, but as there is no global log I can't tell.

I used a file test instead of a directory one as that was easier to do with my original approach. But the `is exists` tests should work with directories as well, I'll give it a try with testing just that.

Reply:
Now, that looks a whole lot better!